### ISSUE TITLE
Histogram bar aria label issue

### ISSUE THREAD

**Issue by toni-sharpe** [toni-sharpe/brainstem-research#38]
### Problem

Histogram bar groupings when generated by the HistogramMaker look for the key, which is a numeric range (in the test, it's groups of 40) and try to find that at `HistogramBarListLabel` in the language key, where it will not be found.

### Solution

Given the dynamic nature of this screen, we cannot know in advance what the label will be so a more thoughtful aria label construction is required. We should be able to get the label in the correct language from the translation for the button clicked then the range number is presented to us already, so a sensible output can be formed.

This can then be tested in the `HistogramMaker` and/or `HistogramBarList` tests.
